.ci/Dockerfile

```diff
@@ -82,6 +83,13 @@ RUN curl -fsSLO "https://github.com/weaveworks/eksctl/releases/download/v${EKSCT
     mv eksctl /usr/local/bin/eksctl && \
     rm eksctl_Linux_amd64.tar.gz
 
+RUN curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 && \
+    chmod 700 get_helm.sh && \ 
+    ./get_helm.sh -v v${HELM_VERSION} --no-sudo && \
+    rm get_helm.sh
+
+RUN helm plugin install https://github.com/quintush/helm-unittest --version 0.2.8
```

Comment:
These new requirements should be mentioned in the `dev-setup.md` file maybe in a new section about developing the Helm charts.

Reply:
👍 Added.  Will get this merged after latest ci tasks finished.